docs/source/user-guide/integrations/airflow.rst

```diff
@@ -0,0 +1,172 @@
+        train_results.save_as_html(os.path.join(dir_path, f'train_integrity_{run_time}.html'))
+        test_results.save_as_html(os.path.join(dir_path, f'test_integrity_{run_time}.html'))
+
+Defining the Model Evaluation Task
+----------------------------------
+
+The :func:`~deepchecks.tabular.suites.model_evaluation` suite will be used to evaluate the model itself.
+It will check for model performance and overfit issues and will save the reports in the ``suite_results`` directory.
+
+.. code-block:: python
+
+    def model_evaluation_step(**context):
+        from deepchecks.tabular.suites import model_evaluation
+        from deepchecks.tabular.datasets.classification.adult import _CAT_FEATURES, _target
+        from deepchecks.tabular import Dataset
+
+        adult_model = joblib.load(context.get("ti").xcom_pull(key="adult_model"))
+        adult_train = pd.read_csv(context.get("ti").xcom_pull(key="train_path"))
+        adult_test = pd.read_csv(context.get("ti").xcom_pull(key="test_path"))
+        ds_train = Dataset(adult_train, label=_target, cat_features=_CAT_FEATURES)
+        ds_test = Dataset(adult_test, label=_target, cat_features=_CAT_FEATURES)
+
+        evaluation_results = model_evaluation().run(ds_train, ds_test, adult_model)
+
+        run_time = datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+        evaluation_results.save_as_html(os.path.join(dir_path, f'model_evaluation_{run_time}.html'))
+
+Creating the DAG
+----------------
+After we have defined all the tasks, we can create the DAG using Airflow syntax. We will define a DAG that will run
+every day.
+
+.. code-block:: python
+
+    with DAG(
+            dag_id="deepchecks_airflow_integration",
+            schedule_interval="@daily",
+            default_args={
+                "owner": "airflow",
+                "retries": 1,
+                "retry_delay": timedelta(minutes=5),
+                "start_date": datetime(2021, 1, 1),
+            },
+            catchup=False,
+    ) as dag:
+        load_adult_dataset = PythonOperator(
+            task_id="load_adult_dataset",
+            python_callable=load_adult_dataset
+        )
+
+        integrity_report = PythonOperator(
+            task_id="integrity_report",
+            python_callable=dataset_integrity_step
+        )
+
+        load_adult_model = PythonOperator(
+            task_id="load_adult_model",
+            python_callable=load_adult_model
+        )
+
+        evaluation_report = PythonOperator(
+            task_id="evaluation_report",
+            python_callable=model_evaluation_step
+        )
+
+    load_adult_dataset >> integrity_report
+    load_adult_dataset >> load_adult_model >> evaluation_report
+
+
+And that's it! In order to run the dag, make sure you place the file in your DAGs folder referenced in your
+``airflow.cfg``. The default location for your DAGs is ``~/airflow/dags``.
+
+After that, you can run the dag using the following command:
```

Comment:
This is a bit confusing, as you wrote that we've scheduled the dag...

Do you mean - the DAG can be also manually triggered (for a single run) by using the following command?
After that you need to initialize (the scheduled) DAG by running the following command...
(whichever - fix phrasing accordingly)